### ISSUE TITLE
Doesn't work on macOS Mojave

### ISSUE THREAD

**Issue by wottpal** [emin/WebPQuickLook#15]
Hey,
I tried to install this extension (both ways), restarted my machine and it doesn't work at all (no thumbs, no quicklook). I tried on two machines running on Mojave - works on none of them :(

Would be great if you provide some help!

Regards from Germany

**Comment by umlearn** [comment]
Same here 😢

**Comment by umlearn** [comment]
@grikomsn Yes, I'm running 10.14.1. Can't figure it out why it doesn't work for me.

**Comment by grikomsn** [comment]
Possible reasons:

- Corrupted or different .webp file: try convert any image using [Squoosh](https://squoosh.app) ([repository](https://github.com/GoogleChromeLabs/squoosh)) and view it using Finder
- Current user can't use the plugin: try install the plugin system-wide using the second command (`install-all.sh`) (I installed using this command)

**Comment by grikomsn** [comment]
@wottpal, @umlearn, any progress?

**Comment by wottpal** [comment]
Hey @grikomsn,
sorry for the late response.

I've tried multiple legit .webp files (also from squoosh) and installed it once again via `install-all.sh` but  no luck 😞

When I fire up QuickLook on a webp file for the first time after installation the window gets large like it wants to show an image but then it jumps right back to just file-info. Some internal error I guess :/

Thanks for your help

**Comment by siliconkibou** [comment]
I also am unable to see the WebP files installing locally or as admin.  I'm on 10.14.2.  I even sanity tested using WebP files from here:  https://developers.google.com/speed/webp/gallery1

**Comment by TomasSestak** [comment]
I have the same issue, will there be a fix?

**Comment by vincenzorm117** [comment]
Im experiencing the same issue 😭

**Comment by grikomsn** [comment]
I'm going to do a fresh install on my MacBook and see if it still works.

**Comment by grikomsn** [comment]
Update: using the 'install all' script still works like a charm on 10.14.3 🎉

**Comment by flekschas** [comment]
I had a similar issue with another QL extension for webp and found that in my case Pixelmator interfered with it. The instructions here https://github.com/Nyx0uf/qlImageSize#limitations solved the issue for me.

**Comment by icefed** [comment]
> I had a similar issue with another QL extension for webp and found that in my case Pixelmator interfered with it. The instructions here https://github.com/Nyx0uf/qlImageSize#limitations solved the issue for me. Maybe this helps here too?

Solved for me too.

**Comment by PontusTideman** [comment]
Tried it out on 10.14.6!

Some times it generates the thumbs for the webp-files, and it works.
Some times it not works on files that just 2 seconds ago worked!
Generates thumbs for animated images, but will not preview them at all!

Take AGES to generate thumbs! And I'm running a 2,8 GHz i7.

**Comment by pot-code** [comment]
uninstall it and install webp(`brew install webp`) instead, works on Mojave 10.14.6

**Comment by rgoldberg** [comment]
> uninstall it and install webp(`brew install webp`) instead, works on Mojave 10.14.6

@pot-code, when you say "works", what do you mean?

I tried uninstalling this QL plugin, and installing the webp brew formula, but I don't see thumbnails or QL previews. I would normally have guessed that the formula would just install libraries, not a QL extension…

**Comment by pot-code** [comment]
As I had installed the plugin, I installed it again by using install-all script, but it didn't work at all. So I did a reboot and uninstalled the plugin via `brew cask uninstall WebPQuickLook`. Then I opened the task manager via `command+option+esc`, select finder and restart it. Bang! everthing now works.

**Comment by meisterleise** [comment]
doesn't work von my Mac, macOS 14.6

**Comment by macmedix** [comment]
Frikken Awesome! Thanks everyone! Works great on MacOS Mojave 10.14.6. 
Select a webp file, tap space bar, view. Yay! Worked 100% on all 24 webp files I happened to have handy in my Downloads folder. 

I installed with:
`curl -L https://raw.github.com/emin/WebPQuickLook/master/install.sh | sh`
and it was done. Did not reboot. 

This Quicklook plugin is So Much Easier than the workarounds I was considering, such as Automator watch folders to watch for any .webp files then calling GraphicConverter to resave as JPG.  

Anyway.... Much appreciated!  THANKS!  :-D
